zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/MappingCreateProcessor.java

```diff
@@ -95,7 +108,7 @@ final var record = command.getValue();
         .ifPresentOrElse(
             existingMapping -> {
               final var errorMessage =
-                  MAPPING_ALREADY_EXISTS_ERROR_MESSAGE.formatted(
+                  MAPPING_SAME_CLAIM_ALREADY_EXISTS_ERROR_MESSAGE.formatted(
```

Comment:
🔧 Let's change this so we get the mapping by id instead of by key, and reject the command with the ID already exists error message.